machina/templates/machina/partials/posted_by.html

```diff
@@ -0,0 +1,59 @@
+{% load forum_member_tags %}
+{% load i18n %}
+
+{% if is_search %}
+    {% if item.poster %}
+        {% url 'forum_member:profile' item.poster as poster_url %}
+        {% blocktrans trimmed with username=item.poster_name creation_date=item.created %}
+            By: <a href="{{ poster_url }}">{{ username }}</a> on {{ creation_date }}
+        {% endblocktrans %}
+    {% else %}
+        {% blocktrans trimmed with poster_username=item.poster_name creation_date=item.created %}
+        By: {{ poster_username }} on {{ creation_date }}
+        {% endblocktrans %}
+    {% endif %}
+{% else %}
```

Comment:
The fact that we need to pass `is_search=True` in order to render this partial templates makes me think that it's probably equivalent to simply let the initial search template untouched. Otherwise would it be possible to make this partial template generic enough so that we don't have to use this `is_search` variable? If the partial is generic enough, it shouldn't make any assumption regarding the context in which it is used. 🤷‍♂️

Reply:
Hi I had a second look at it, IMO, we should pass through as we did before (before the fixup), the username, the creation date and poster now that we have 3 cases : topics, posts and search, this way the partial will be generic.
Before I go for it, I prefer to have your feedback.
Thank you

Reply:
We can try! Though I think the resulting partial should be as simple as possible and not contain too much logic.